fe/fe-core/src/main/java/com/starrocks/catalog/RangePartitionInfo.java

```diff
@@ -75,7 +75,7 @@ public class RangePartitionInfo extends PartitionInfo {
     @SerializedName(value = "partitionColumns")
     private List<Column> partitionColumns = Lists.newArrayList();
     // formal partition id -> partition range
-    private Map<Long, Range<PartitionKey>> idToRange = Maps.newHashMap();
+    protected Map<Long, Range<PartitionKey>> idToRange = Maps.newHashMap();
```

Comment:
do you support temp partitions?

Reply:
This is a good question. I think may be next PR will consider operations such as adding and deleting partitions. At the same time, it should theoretically support temporary partitions.